src/richie-front/js/components/CourseGlimpse/CourseGlimpse.tsx

```diff
@@ -35,22 +35,25 @@ export const CourseGlimpse = injectIntl(
     const { course, intl, organizationMain } = props;
 
     return (
-      <div className="course-glimpse-container">
-        <div className="course-glimpse">
+      <a className="course-glimpses__item course-glimpses__item--link" href="#">
```

Comment:
I don't think this name change is a good idea.

A `course-glimpse` is already an item that fits in a `course-glimpse-list` (or maybe just in a generic kind of list). To me these are two different components with two different purposes and therefore non-derived names.

`course-glimpse` => `course-glimpses__item` does not make a lot of sense for a way to name a single `CourseGlimpse`.